### PR TITLE
focus to buttons on dashboard view

### PR DESCRIPTION
**Pull request by vanpertsch**
Before | After
---|---
Screenshot 2022-03-10 at 15 31 26 | Screenshot 2022-03-10 at 14 16 56
Screenshot 2022-03-10 at 15 31 35 | Screenshot 2022-03-10 at 14 17 11
Screenshot 2022-03-10 at 15 32 05 | Screenshot 2022-03-10 at 14 17 20

Quick fixes for the buttons in the dashboard view. 
Alternatively, we could change the status button using the new Vue button component, but since there's a lot of functionality, etc., I decided to go with the quick option.
The weather button is a Vue action button. Some custom styles were already there, so I thought the focus might be quick to change there too. (If the action buttons get a different focus style, it needs to be reverted).
The changes also take effect for the customize button. 

@skjnldsv do you think this is a way to go?

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]
Code looks good, not sure if this should be the intended visual @jancborchardt 

Btw is 2px for the focus indicator expected? According to https://www.sarasoueidan.com/blog/focus-indicators/ (good read btw if you can afford to spend one hour on it) it can also be 1px as long.

>  The contrasting area is at least as large as the area of a 1px thick perimeter of the unfocused component…

File: apps/user_status/src/UserStatus.vue [hunk 1]
```diff
@@ -297,6 +297,9 @@ export default {
 			&:focus {
 				background-color: var(--color-background-hover);
 			}
+			&:focus {
```
Comment (PVince81) [inline apps/user_status/src/UserStatus.vue:300]: reuse the above block ?

Comment (MichaIng) [reply apps/user_status/src/UserStatus.vue:300]: That would add the border to `active` and `hover` as well, which, I guess, isn't wanted.

**Review by skjnldsv — APPROVED** [review]
If the design team is :ok_hand:

File: apps/dashboard/src/App.vue [hunk 2]
```diff
@@ -543,11 +543,15 @@ export default {
...
 	backdrop-filter: var(--background-blur);
+	opacity: 1 !important;
 
 	&:hover,
 	&:focus,
 	&:active {
-		background-color: var(--color-background-hover);
+		background-color: var(--color-background-hover)!important;
+	}
+	&:focus {
```
Comment (CarlSchwan) [inline apps/dashboard/src/App.vue:553]:
```suggestion
	&:focus-visible {
```

See https://pawelgrzybek.com/the-difference-between-css-focus-and-focus-visible-pseudo-class/

Comment (vanpertsch) [reply apps/dashboard/src/App.vue]: thx @CarlSchwan  :)

**Comment by vanpertsch** [comment]
@nimishavijay @jancborchardt do you approve the focus changes for dashboard?

**Pull request merged**